media/components/analysis/index.html

```diff
@@ -137,194 +121,9 @@
         </div>
     </div>
 
+    <script src="{{ SCRIPT_SRC }}/analysis.js"></script>
```

Comment:
All `ts` files are compiled down and packed into a set of `js` dist files. It is, however, not just one large file, but instead one file per component. It could be achieved with one monolithic file, but in order to make web components load as little JS as necessary, I have organized it that each component (including the extension itself) are contained in one dist file each. This means that there are the following dist files in the end:
1. `extension.js` -> Extension source
2. `analysis.js` -> Source for the analysis web component
3. `breakpoints.js` -> Source for the breakpoints list web component
4. `transformationHistory.js` -> Dito for the history
5. `transformations.js` -> Dito for the xform list
6. `outline.js` -> Dito for the outline

This strikes a balance between reducing the amount of `js` files (from multiple 100s to 6) and still keeping individual files as minimalistic as possible. If the outline component would need to load the entire source, it would take noticeably longer to appear.

Reply:
That's great, needs to be documented somewhere